add blueSky color-setting

sanskruthiya opened the pull request:
This pull request adds a new color theme named "skyBlue".
It provides users with the option to select a standard clear blue sky color.

JinIgarashi reviewed (commented):
@sanskruthiya Thank you for the PR! this new sky color looks very good, I like it. How about replacing current `solarNoon` color to this new color other than adding new one?

`solarNoon` will be the standard color of this plugin, and currently it shows just default values from maplibre. I think your sky color is much better.

JinIgarashi commented:
@sanskruthiya Looks great! could you fix lint error? I think `pnpm format` can be done to fix formatting

sanskruthiya commented:
@JinIgarashi Thank you for your comment. I have replaced the new color theme "blueSky" to the default, and removed blueSky and old default setting. Please check it, and I hope you like it!
